src/portfolio.js

```diff
@@ -18,16 +18,47 @@ const greeting = {
 	portfolio_repository: "https://github.com/ashutosh1919/masterPortfolio",
 };
 
-const socialMediaLinks = {
+const socialMediaLinks = [
 	/* Your Social Media Link */
-	github: "https://github.com/ashutosh1919",
-	linkedin: "https://www.linkedin.com/in/ashutosh-hathidara-88710b138/",
-	gmail: "ashutoshhathidara98@gmail.com",
-	gitlab: "https://gitlab.com/ashutoshhathidara98",
-	facebook: "https://www.facebook.com/laymanbrother.19/",
-	twitter: "https://twitter.com/ashutosh_1919",
-	instagram: "https://www.instagram.com/layman_brother/"
-};
+	// github: "https://github.com/ashutosh1919",
+	// linkedin: "https://www.linkedin.com/in/ashutosh-hathidara-88710b138/",
+	// gmail: "ashutoshhathidara98@gmail.com",
+	// gitlab: "https://gitlab.com/ashutoshhathidara98",
+	// facebook: "https://www.facebook.com/laymanbrother.19/",
+	// twitter: "https://twitter.com/ashutosh_1919",
+	// instagram: "https://www.instagram.com/layman_brother/"
+  
+	{
+		name: "Github",
+		link: "https://github.com/ashutosh1919",
+		className: "github",
+		fontAwesomeIcon: "fa-github", //For more details check out the FontAwesome icons list
+	  },
+	  {
+		name: "LinkedIn",
+		link: "https://www.linkedin.com/in/ashutosh-hathidara-88710b138/",
+		className: "linkedin",
+		fontAwesomeIcon: "fa-linkedin-in", //For more details check out the FontAwesome icons list
+	  },
+	  {
+		name: "Gmail",
+		link: "mailto:ashutoshhathidara98@gmail.com",
+		className: "google",
+		fontAwesomeIcon: "fa-google", //For more details check out the FontAwesome icons list
+	  },
+	  {
+		name: "Twitter",
+		link: "https://twitter.com/ashutosh_1919",
+		className: "twitter",
+		fontAwesomeIcon: "fa-twitter", //For more details check out the FontAwesome icons list
+	  },
+	  {
+		name: "Facebook",
+		link: "https://www.facebook.com/laymanbrother.19/",
+		className: "facebook",
+		fontAwesomeIcon: "fa-facebook-f", //For more details check out the FontAwesome icons list
+	  },
```

Comment:
Why not to add instagram? If there is a problem like specific fontAwesomeIcon is not present then we need to validate it and shouldn't render it but we need to give flexibility to add icons because user may mess this up by adding so much of junk.

Reply:
This is one of the reasons, I wanted to go with rendering icons if the link is provided.
We can suggest what other accounts are available are available in the documentation.

Reply:
Yeah. I observed that some icons are missing the support of the font-awesome icons list. One thing we can do is to increase the icon support. When I was building my own portfolio using masterfolio, I found `iconify` library [https://iconify.design/icon-sets/](url) more helpful (which is already in the project) to add various other icons and there is a customization for some styling as well. So, what are your thoughts on this? Should we modify the existing views using `iconify`? or do you have anything in mind?

Reply:
> This is one of the reasons, I wanted to go with rendering icons if the link is provided.
> We can suggest what other accounts are available are available in the documentation.

Yeah @th3c0d3br34ker we can even do that as well if we want to support only a limited set of icons and give a list of supported icons

Reply:
@dineshnadimpalli, I don't think we need to consider adding iconify as of now. Only one icon is remaining and that is instagram which is present in Fontawesome. Also, if you take a look at the css of it, then I have also defined icon color for it. So, please add the icon and we are good to go. https://fontawesome.com/v4.7.0/icon/instagram